crates/orchestrator/src/jobs/proving_job/mod.rs

```diff
@@ -128,6 +128,9 @@ impl Job for ProvingJob {
                 Ok(JobVerificationStatus::Pending)
             }
             TaskStatus::Succeeded => {
+                // TODO: call isValid on the contract over here to cross-verify whether the proof was registered on
```

Comment:
won't be generic as of now, we would have tech debt because currently we are making a client of l1 contracts but in future given we have l3 app chains as well, we would need to make it generic so it's able to call the starknet contracts to validate as well

Reply:
kept as is